packages/components/app/styles/components/table.scss

```diff
@@ -0,0 +1,177 @@
+//
+// TABLE
+//
+//
+$hds-table-border-radius: 6px;
+$hds-table-border-width: 1px;
+$hds-table-inner-border-radius: $hds-table-border-radius - $hds-table-border-width;
+$hds-table-border-color: var(--token-color-border-primary);
+$hds-table-cell-padding-medium: 12px 16px;
+$hds-table-cell-padding-short: 4px 16px;
+$hds-table-cell-padding-tall: 20px 16px;
+
+.hds-table,
+.hds-table-sortable {
+  width: 100%;
+  font-size: var(--token-typography-body-200-font-size);
+  font-family: var(--token-typography-font-stack-text);
+  line-height: var(--token-typography-body-200-line-height);
+  table-layout: fixed;
+  border: $hds-table-border-width solid $hds-table-border-color;
+  border-radius: $hds-table-border-radius;
+  border-spacing: 0;
+}
+
+// TABLE HEADER
+.hds-table__thead {
+  .hds-table__tr {
+    color: var(--token-color-foreground-strong);
+    background-color: var(--token-color-surface-strong);
```

Comment:
Ah good catch! I'm going to take them off the th instead though, since they're on the tr in the tbody and it's also where folks generally expect to find these things (esp. re: striping)